tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/BaseWhenUsingApiUnitTests.cs

```diff
@@ -11,25 +12,59 @@ public abstract class BaseWhenUsingApiUnitTests : IDisposable
     protected readonly HttpClient? Client;
     private readonly CustomWebApplicationFactory? _webAppFactory;
     private readonly bool _initSuccessful;
+    public static string BearerTokenSigningKey;
 
     protected BaseWhenUsingApiUnitTests()
     {
         try
         {
+            var configuration = new ConfigurationBuilder()
+                .AddUserSecrets<Program>()
+                .Build();
+
             _webAppFactory = new CustomWebApplicationFactory();
             _webAppFactory.SetupTestDatabaseAndSeedData();
 
             Client = _webAppFactory.CreateDefaultClient();
             Client.BaseAddress = new Uri("https://localhost:7128/");
 
             _initSuccessful = true;
+
+            BearerTokenSigningKey = IsRunningLocally(configuration)
+                ? configuration["GovUkOidcConfiguration:BearerTokenSigningKey"] : "StubPrivateKey123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
Can this not just check if the configuration has a value for BearerTokenSigningKey and fallback when it doesn't?

Even better build this into the ConfigurationBuilder:
```csharp
.AddUserSecrets<Program>()
.AddInMemoryCollection(
    new Dictionary<string, string?> {
        {"GovUkOidcConfiguration:BearerTokenSigningKey", "StubPrivateKey123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"},
    }
);
```

Reply:
Much better, thanks!
(Had to swap the order, as the value is picked up from the most recently added provider with the key.)